lgc/state/PipelineState.cpp

```diff
@@ -1003,8 +1003,7 @@ PipelineState::findResourceNode(ResourceNodeType nodeType, unsigned descSet, uns
       return {&node, &node};
   }
 
-  if (nodeType == ResourceNodeType::DescriptorFmask &&
-      getOptions().shadowDescriptorTable != ShadowDescriptorTableDisable) {
+  if (nodeType == ResourceNodeType::DescriptorFmask && getOptions().enableFmask) {
```

Comment:
Pay attention to the comment inside of the `if` statement, please. This should not be changed, or the comment also needs changing.